tests/apps/schedulerapp/.cache-include

```diff
@@ -0,0 +1 @@
+../utils/*.go
```

Comment:
nit: end file in newline

Reply:
none of the other .cache-include files have a newline from what I saw